ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -1,54 +1,78 @@
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "ContactTracing" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/ContactTracing?id=@Model.NotificationId">Contact Tracing</a>
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "SocialRiskFactors" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/SocialRiskFactors?id=@Model.NotificationId">Social Risk Factors</a>
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "Travel" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Travel?id=@Model.NotificationId">Travel/visitor history</a>
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "Comorbidities" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Comorbidities?id=@Model.NotificationId">Co-morbidities</a>
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "Immunosupression" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/Immunosupression?id=@Model.NotificationId">Immunosupression</a>
+                                </li>
+                                <li class="@("contents-list-item " + (currentPage == "PreviousHistory" ? "contents-list-item--current" : ""))">
+                                  <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/Notifications/Edit/PreviousHistory?id=@Model.NotificationId">Previous History</a>
+                                </li>
+                            </div>
+                            <div style="width: 100%"> 
+                                @RenderBody()
+                            </div>
+                        </div>
+                    </nhs-grid-column>
+                </nhs-grid-row>
+            </nhs-main-wrapper>
+        </nhs-width-container>
     </div>
+    <div class="nhsuk-main-wrapper notification-sideline-container">
+        <aside class="manage-notification-section">
+            <h2 class="govuk-heading-m" id="manage-notification-title">
+                Manage Notification
+            </h2>
+            <form method="post">

```

Comment:
Like we talked about yesterday, we'll have to expand this out to full be a single form